frontend/src/metabase/nav/containers/MainNavbar/tests/common.unit.spec.tsx

```diff
@@ -122,27 +122,27 @@ describe("nav > containers > MainNavbar", () => {
   describe("browse models link", () => {
     it("should render when there are models", async () => {
       await setup({ models: [createMockModelResult()] });
-      const listItem = await screen.findByRole("listitem", {
+      const listItem = screen.getByRole("listitem", {
         name: /Browse models/i,
       });
-      const link = await within(listItem).findByRole("link");
+      const link = within(listItem).getByRole("link");
       expect(link).toBeInTheDocument();
       expect(link).toHaveAttribute("href", "/browse/models");
     });
 
-    it("should render when there are no models", async () => {
+    it("should not render when there are no models", async () => {
```

Comment:
We changed the behavior in v51.
The fact that I had to split `MainNavbar.unit.spec.tsx` into three separate files and then backport them didn't help in catching this. CI did.